### ISSUE TITLE
Can I delete account to make channel with same name?

### ISSUE THREAD

**Issue by maduce** [keybase/keybase-issues#3415]
I made an account with the name i wanted for a Team name. The Keybase client says we cannot have a team name with the same name as an existing user. If I delete the user from the keybase site can I create a team with that name? I do not want to test this because Keybase.io says once deleted you cannot create that account again.

Side note; It seems as keybase gets more popular the number of team names available may become limited due to people picking them as user names. Is this _no conflict_ policy going to change soon?

**Comment by mlsteele** [comment]
Nope, once a user is created that name cannot be used for a team. Not even if the team is deleted. I know this can be annoying.

> Keybase.io says once deleted you cannot create that account again
Maybe we can clarify the wording here to make it clear that the name also cannot be used as a team name. Where did you see this?

**Comment by maduce** [comment]
The message about the user/team name conflict shows up when you try to create the team which happens after you create an account. It would have been great if that message was posted as a warning in ```keybase signup```, somewhere on the install page, somewhere in the installation documentation where it explains how to create an account or in ```Command line basics```. I do not recall this conflict information being on the Teams blog post either.

Those surprised by this annoyance begin the keybase experience requiring a permanent workaround with an unplanned team name. I look forward to a keybase type solution where this conflict is resolvable.

**Comment by junderw** [comment]
I agree that there should be a warning on signup.

However, I believe allowing multiple users and teams to reclaim the same name once deleted will be a security risk.

Keybase is the best mix of key exchange integrity and usability.

One of the things that was sacrificed on the usability side is reclaiming deleted namespaces.

**Comment by maduce** [comment]
This might sound like nonsense but maybe there is a way to allow users to turn an account into a team. Then there is no deleting or reclaiming names.

**Comment by junderw** [comment]
That is true. Github has a way for users to convert to organizations (but not the other way around, so there are big warnings when you try to do it.)

Perhaps making a new Issue with a feature request stating "Allow a user to convert itself into a team." would be ok.
